app/controllers/trainees/itt_start_dates_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Trainees
+  class IttStartDatesController < ApplicationController
+    before_action :authorize_trainee
+
+    def edit
+      @start_form = IttStartDateForm.new(trainee)
```

Comment:
Suggest renaming `@start_form` to `@itt_start_date_form` as this is the common approach in other controllers that use form objects i.e. naming the variable exactly the same as the form name.

Reply:
fixed